src/taco-cli/cli/utils/kitHelper.ts

```diff
@@ -102,7 +102,8 @@ class KitHelper {
 
     private static acquireKitPackage(): Q.Promise<ITacoKits> {
         if (!KitHelper.kitPackagePromise) {
-            KitHelper.kitPackagePromise = TacoPackageLoader.lazyTacoRequire<ITacoKits>(KitHelper.TACO_KITS_NPM_PACKAGE_NAME, KitHelper.dynamicDependenciesLocation, tacoUtility.InstallLogLevel.taco);
+            KitHelper.kitPackagePromise = TacoPackageLoader.lazyTacoRequire<ITacoKits>(KitHelper.TACO_KITS_NPM_PACKAGE_NAME,
+                KitHelper.dynamicDependenciesLocation, tacoUtility.InstallLogLevel.error);
```

Comment:
shouldn't the logleve be here taco? so that we get "taco" messages like Downloading etc.?